### PR TITLE
factor out sc_isolation_protocols_io_doi

### PR DESCRIPTION
**Pull request by mccalluc**
Use the suggest-edit mechanism to indicate which is the correct definition for this field; Chuck will followup and regenerate docs.

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# scrnaseq-hca-v0.yaml
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml:1]:
```suggestion
# scrnaseq-hca-v0.yaml; scrnaseq-v0.yaml; scrnaseq-v1.yaml; scrnaseq-v2.yaml; scatacseq-v0.yaml; scatacseq-v1.yaml
```

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    cells separated into a single-cell suspension?'
+  type: string
+
+# scrnaseq-v0.yaml; scrnaseq-v1.yaml; scrnaseq-v2.yaml
+- name: sc_isolation_protocols_io_doi
+  description: "Link to a protocols document answering the question: How were single cells separated into a single-cell suspension?"
+
+# scatacseq-v0.yaml; scatacseq-v1.yaml
+- name: sc_isolation_protocols_io_doi
+  description: 'Link to a protocols document answering the question: How were single cells separated into a single-cell suspension?'
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml:19]:
```suggestion
```

**Review by cebriggs7135 — COMMENTED** [review]
Since all of the following assays have the same DOI pattern requirement as that shown for HCA, I consolidated them: 

scrnaseq-hca-v0.yaml; scrnaseq-v0.yaml; scrnaseq-v1.yaml; scrnaseq-v2.yaml; scatacseq-v0.yaml; scatacseq-v1.yaml.

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml (hunk 1 above)
Comment (mccalluc) [reply src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml:1]: No need to update this list: It's only there to help your review, but I'm removing it during the merge.

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# scrnaseq-hca-v0.yaml
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml:1]:
```suggestion
```

**Pull request closed without merging**